src/controls/SelectTeamChannelPicker/constants.ts

```diff
@@ -0,0 +1 @@
+export const  TEAMS_SVG_LOGO  =   "https://static2.sharepointonline.com/files/fabric-cdn-prod_20200430.002/assets/brand-icons/product/svg/teams_48x1.svg";
```

Comment:
I think to be on a save side of things we need to use [https://static2.sharepointonline.com/files/fabric/assets/brand-icons/product/svg/teams_48x1.svg](https://static2.sharepointonline.com/files/fabric/assets/brand-icons/product/svg/teams_48x1.svg) as icon URL. See [this documentation](https://developer.microsoft.com/en-us/fluentui#/styles/web/office-brand-icons)